atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -97,9 +117,22 @@ private AtlasDbHttpClients() {
                 .errorDecoder(errorDecoder)
                 .client(client)
                 .retryer(failoverFeignTarget)
+                .options(feignOptions)
                 .target(failoverFeignTarget);
     }
 
+    @VisibleForTesting
+    static <T> T createProxyWithFailoverWithoutBackoff(
+            Optional<SSLSocketFactory> sslSocketFactory, Collection<String> endpointUris, Class<T> type) {
+        Request.Options options = new Request.Options(ONE_MIILISECOND, ONE_MIILISECOND);
```

Comment:
It's a fair point, we might be able to play with this a little bit to get more out of the tests. I think that's a bit too involved for now though, since the tests do seem to be working and giving us signal.

I agree with you about the name! Struggled a little here; I've gone for `createProxyWithQuickFailoverForTesting`. What do you think?